Makefile

```diff
@@ -25,3 +25,8 @@ docs:
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
 	open docs/_build/html/index.html
+
+docker-build:
+	@docker build \
```

Comment:
For `make docker-build` to work I needed to add  `-t viper .`, @mbeacom does it work without that for you?
```
@docker build -t viper . \
		--build-arg VCS_REF=`git rev-parse --short HEAD` \
````

Reply:
@DavidKnott Yes, it seems to work fine on my end:

```
~/C/viper   slim-docker  make docker-build           Sat Dec 23 14:58:37 2017
Sending build context to Docker daemon  2.009MB
Step 1/8 : FROM python:3.6-slim
3.6-slim: Pulling from library/python
c4bb02b17bb4: Already exists
```

I can add the tag to it, though!  Wasn't sure if we wanted to tag it or not.